packages/amplify-migration-tests/custom-resources/custom-cdk-stack-vLatest.package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "custom-resource",
+    "version": "1.0.0",
+    "description": "",
+    "scripts": {
+        "build": "tsc",
+        "watch": "tsc -w",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "dependencies": {
+        "@aws-amplify/cli-extensibility-helper": "^3.0.0-beta.0",
+        "aws-cdk-lib": "~2.53.0",
+        "constructs": "^10.0.5"
```

Comment:
Halfway into the migration test for custom stacks, we simulate the customer migrating from v10 to vLatest.
In doing so, they must also update the package.json file to use CDKv2 dependencies.
The contents of this file are used to simulate the effective change that customers would perform during the migration.
Notably, migrating to aws-cdk-lib, and updating the extensibility helper version.